lib/ret/avatar.ex

```diff
@@ -93,6 +94,29 @@ defmodule Ret.Avatar do
     |> avatar_to_collapsed_files()
   end
 
+  def version(%Avatar{} = avatar) do
+    avatar.updated_at |> NaiveDateTime.to_erl |> :calendar.datetime_to_gregorian_seconds
+  end
+
+  def url(%Avatar{} = avatar) do
```

Comment:
nit: you can re-write these to be one line:

`def url(%Avatar{} = avatar), do: "#{RetWeb.Endpoint.url()}/api/v1/avatars/#{avatar.avatar_sid}"`